doc/src/Run_basics.rst

```diff
@@ -2,15 +2,22 @@ Basics of running LAMMPS
 ========================
 
 LAMMPS is run from the command line, reading commands from a file via
-the -in command line flag, or from standard input.
-Using the "-in in.file" variant is recommended:
+the -in command line flag, or from standard input.  Using the "-in
+in.file" variant is recommended.  The name of the LAMMPS executable is
+either ``lmp`` or ``lmp_<machine>`` with `<machine>` being the machine
+string used when compiling LAMMPS.  This is required when compiling
+LAMMPS with the traditional build system (e.g. with ``make mpi``),
+but optional when using CMake to configure and build LAMMPS:
```

Comment:
May want to explain why `-in in.file` is preferred: 

In https://docs.lammps.org/Run_basics.html it has this note:


`The redirection operator “<” will not always work when running in parallel with mpirun; for those systems the -in form is required.`

Reply:
Actually this is in the same section (didn't realize that), so could add something like `see note below`.

Reply:
If it was my choice we would drop that entirely. This is effectively a leftover from the Fortran days when there was no portable way to process command line flags and thus people used I/O redirection to read input files. No longer supporting I/O redirection would make some of the more convoluted code in the LAMMPS class significantly simpler. But some people's habits are hard to break, especially if they go back > 25 years. ;-)